packages/suite-native-settings/src/components/DeviceSettings.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { SettingsSection } from './SettingsSection';
+import { SettingSectionItem, SettingsSectionItem } from './SettingsSectionItem';
+
+const deviceSettingItems: SettingSectionItem[] = [
+    { title: 'Customization', description: 'Name, Homescreen', iconName: 'palette' },
+    { title: 'Security', description: 'PIN Active, Passphrase enabled', iconName: 'lock' },
+    { title: 'Danger Area', description: 'Factory reset, Custom firmware', iconName: 'lock' },
+];
```

Comment:
`as const`

Reply:
It's already typed so it shouldn'n be needed to cast it into const. Or why do you think it's good?